plugins/modules/perlmod_install_info.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Jonathan Kamens <jik@kamens.us>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# pylint: disable=missing-module-docstring
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: perlmod_install_info
+
+short_description: Determine from where to install Perl modules
+
+requirements:
+    - C(apt-file) executable in search path on systems that use the apt package
+      manager
+    - C(dnf) or C(yum) executable in search path on systems that use the
+      dnf/yum package manager
+    - C(cpanm) executable in search path if you want to be able to search for
+      packages using cpanm
+
+version_added: "7.0.0"
+
+description:
+    - Searches dnf, yum, apt, and/or cpanm to determine the best source from
+      which to install a Perl module.
+    - Prefers the OS repositories over cpanm. Only returns modules from CPAN
+      when it is searchable (I(try_cpan) is C(auto) or C(true) and the C(cpanm)
+      executable is available) and the returned modules were not found
+      in any other source.
```

Comment:
This should (also) be mentioned in the return value's `description`.

Reply:
I am confused. I had details in the return value's description before and you asked me in a comment to simplify the return value's description and move the information into the description of the module, or at least, that's what I thought you were suggesting. Since we've gone back and forth several times about this, please just tell me what you think the return value description should say so that there's no ambiguity.

Reply:
No, that was not in `description:` of the return value, but in `returned:` of the return value. These are two very different fields. `description:` can have long texts with multiple paragraphs, while `returned:` should be rather short.